newsroom/web/default_settings.py

```diff
@@ -366,6 +366,9 @@
     'analyze_wildcard': False
 }
 
+# count above 10k
+ELASTICSEARCH_TRACK_TOTAL_HITS = True
```

Comment:
I wonder how this would affect customers with large number of documents & large queries.
What are your thoughts on leaving default to 10k, and changing this to `True` per customer?
In the UI, if the count is `10,000` then maybe we show `10,000+` to indicate there are more (if the customer(s) are okay with that experience)

Reply:
I did some testing locally and didn't see any effect so would keep it on by default with the option to turn it off if needed.
We have it on in superdesk and it was never an issue afaik.

Reply:
Sounds good